high_card.rb

```diff
@@ -1,5 +1,31 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 ]
```

Comment:
fine to do that this time! if you have more time, I'd consider trying to write in the actual starter code again and play it out